Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -176,8 +176,13 @@ static void launchWithExtensionKit(ProcessLauncher& processLauncher, ProcessLaun
                 [process invalidate];
                 return;
             }
-            launcher->m_xpcConnection = [process makeLibXPCConnectionError:nil];
+            NSError* error = nil;
+            launcher->m_xpcConnection = [process makeLibXPCConnectionError:&error];
             launcher->m_process = WTFMove(process);
+            if (!launcher->m_xpcConnection) {
+                NSLog(@"Failed to create XPC connection, error = %@", error);
```

Comment:
Does it make sense to keep the `_SEExtensionProcess` if we can't get an XPC connection, and return without finishing the process launch?

Reply:
That is a good point! I removed this part of the patch, since this part was already changed by another commit.